scripts/scil_fodf_visualize.py

```diff
@@ -233,11 +242,11 @@ def _get_data_from_inputs(args):
         if args.peaks_values:
             assert_same_resolution([args.peaks_values, args.in_fodf])
             peak_vals =\
-                nib.nifti1.load(args.peaks_values).get_fdata(dtype=np.float32)
+                nib.load(args.peaks_values).dataobj
             data['peaks_values'] = peak_vals
     if args.variance:
         assert_same_resolution([args.variance, args.in_fodf])
-        variance = nib.nifti1.load(args.variance).get_fdata(dtype=np.float32)
+        variance = nib.load(args.variance).dataobj
```

Comment:
Usually in other scripts, we sill use get_fdata. Can you tell me the difference?

Reply:
The use of `dataobj` makes it possible to `lazy load` the volumes instead of putting them all in memory. I changed some loadings while testing memory problems in the viz module. I'll revert this back, but we should have a discussion on what should be `lazy loaded` in Scilpy, and make use of the `volume_iterator` in `scilpy.image.utils`to do the job